src/types/omit.ts

```diff
@@ -0,0 +1 @@
+export type DistributiveOmit<T, K extends keyof any> = T extends any ? Omit<T, K> : never;
```

Comment:
What's the purpose of this?

Reply:
This is used to omit on union types. Without this, the type that `Omit` produces is too simplified in some cases.